Lab_1_5/Lab_1_5/Program.cs

```diff
@@ -0,0 +1,277 @@
+                            default:
+                                Console.WriteLine("Exit");
+                                break;
+                        }
+
+                    }
+                    catch (Exception e)
+                    {
+                        Console.WriteLine("Error"+e.Message);
+                    }                   
+                    Console.ForegroundColor = ConsoleColor.Green;
+                    Console.WriteLine("Press Spacebar to exit; press any key to continue");
+                    Console.ForegroundColor = ConsoleColor.White;
+                }
+                while (Console.ReadKey().Key != ConsoleKey.Spacebar);
+            }
+            catch (Exception ex)
+            {
+                Console.WriteLine(ex.Message);
+            }
+        }
+        #region ToFromBinary
+        static void My_Binary()
+        {
+            //Implement positive integer variable input
+            Console.WriteLine("Enter a number:");
+            var input = Console.ReadLine();
+            var enteredNumber = uint.Parse(input);
+
+            //Present it like binary string
+            //   For example, 4 as 100
+            //Use modulus operator to obtain the remainder  (n % 2) 
+            //and divide variable by 2 in the loop
+
+            
+            var binaryString = "";
+
+            while (enteredNumber > 0)
+            {
+                var oneZero = enteredNumber % 2;
+                binaryString += oneZero;
+                enteredNumber = enteredNumber / 2; 
+            }
+
+            //Use the ToCharArray() method to transform string to chararray
+            //and Array.Reverse() method
+            var charArray = binaryString.ToCharArray();
+            Array.Reverse(charArray);
+
+            Console.WriteLine(charArray);
+        }
+        #endregion
+
+        #region ToFromUnary
+        static void My_strings()
+        {
+            //Declare int and string variables for decimal and binary presentations
+            //Implement two positive integer variables input
+
+            Console.WriteLine("Enter first number:");
+            var input = Console.ReadLine();
+
+            var firstNumber = uint.Parse(input);
+
+            Console.WriteLine("Enter second number:");
+            input = Console.ReadLine();
+
+            var secondNumber = uint.Parse(input);
+
+            //To present each of them in the form of unary string use for loop
+            var firstStringRepresentation = GetUnaryStringPreserntation(firstNumber);
+            var secondStringRepresentation = GetUnaryStringPreserntation(secondNumber);
+
+            //Use concatenation of these two strings 
+            //Note it is necessary to use some symbol ( for example “#”) to separate
+            var resultString = firstStringRepresentation + "#" + secondStringRepresentation;
+
+            Console.WriteLine(resultString);
+
+            //Check the numbers on the equality 0
```

Comment:
lets remove all such comments then will be easier to review PR

Reply:
Fixed